client/src/Components/CommonComponents/OfferCard/style.css

```
@@ -0,0 +1,161 @@
+.offer-card {
+  margin: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  color: rgba(255, 255, 255, 0);
+  cursor: pointer;
+}
+  
+.offer-card:hover {
+  box-shadow:  0 0 30px -15px rgba(0, 0, 0, 0.3)
+}
+
+.offer-card--header {
+  display: flex;
+}
+
+.offer-card__description{
+  line-height: 1.5rem;
+  letter-spacing: 0.5px;
+}
+
+.not_active {
+  display: none;
+}
+
+.offer-card__position {
+  color: #0F9CAF;
+  font-size: 1.2rem;
+}
+
+.offer-card__title {
+  color: rgba(51, 51, 51, 0.7);
+  font-size: 0.9rem;
+}
+
+.offer-card__favourite  {
+  position: absolute;
+  top: 1px;
+  right: 11px;
+  font-size: 22px;
+  color: rgba(134, 133, 133, 0.2);
+  pointer-events: none;
+}
+
+.offer-card__favourite:hover  {
+  color: rgba(134, 133, 133, 0.6);
+}
+
+.offer-card__description,
+.offer-card__favourite:hover {
+  cursor: pointer;
+}
+
+.offer-card__favourite--saved {
+  color: #0F9CAF;
+}
+
+.offer-card__save-btn,
+.offer-card__save-btn:active {
+  background: none !important;
+  border: none !important;
+  position: absolute !important;
+  top: -1px !important;
+  right: 11px !important;
+  padding-top: 0 !important;
+  height: 20px;
+  width: 30px;
+}
+
+.card-hovered:hover {
```

hovered is a modifier 
```suggestion
.offer-card--hovered:hover {
```